src/NuGet.Indexing/ResponseFormatter.cs

```diff
@@ -346,7 +346,19 @@ public static void WriteStatsResult(JsonWriter jsonWriter, NuGetIndexSearcher se
             jsonWriter.WriteStartObject();
             WriteProperty(jsonWriter, "numDocs", searcher.IndexReader.NumDocs());
```

Comment:
@shishirx34 and I spoke offline and resolved our confusions. I was specifically referring to the list of properties at the top of the file, not the part where we construct a series of objects.